docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,221 @@
+            - determine the UEF based on the draw pattern for HIGH: `if draw_pattern == "HIGH": expected_uef = 0.6920 – (0.0013 × storage_volume)`
+            - check whether the uniform_energy_factor is valid: `if uniform_energy_factor != NULL
+              - check if the actual uniform_energy_factor is >= to the expected: `if uniform_energy_factor >= expected_UEF:`
+                - set rule_status to "PASS": `rule_status = "PASS"`
+              - otherwise, set rule status to FAIL and provide the note that the expected UEF is less than the required UEF: `else: rule_status = "FAIL"; rule_note = "The modeled Uniform energy Factor (UEF - " + uniform_energy_factor + ") is less than required (" + expected_UEF + ")"`
+            - otherwise, set rule status to UNDETERMINED and provide a note: `else: rule_status = "UNDETERMINED"; rule_note = "Uniform Energy Factor is required but not provided"`
+          - otherwise, if the storage volume is between 55 and 100 gallons: `elif storage_volume <= 100:`
+            - determine the UEF based on the draw pattern for VERY_SMALL: `if draw_pattern == "VERY_SMALL": expected_uef = 0.6470 – (0.0006 × storage_volume)`
+            - determine the UEF based on the draw pattern for LOW: `if draw_pattern == "LOW": expected_uef = 0.7689 – (0.0005 × storage_volume)`
+            - determine the UEF based on the draw pattern for MEDIUM: `if draw_pattern == "MEDIUM": expected_uef = 0.7897 – (0.0004 × storage_volume)`
+            - determine the UEF based on the draw pattern for HIGH: `if draw_pattern == "HIGH": expected_uef = 0.8072 – (0.0003 × storage_volume)`
+            - check whether the uniform_energy_factor is valid: `if uniform_energy_factor != NULL
+              - check if the actual uniform_energy_factor is >= to the expected: `if uniform_energy_factor >= expected_UEF:`
+                - set rule_status to "PASS": `rule_status = "PASS"`
+              - otherwise, set rule status to FAIL and provide the note that the expected UEF is less than the required UEF: `else: rule_status = "FAIL"; rule_note = "The modeled Uniform energy Factor (UEF - " + uniform_energy_factor + ") is less than required (" + expected_UEF + ")"`
+            - otherwise, set rule status to UNDETERMINED and provide a note: `else: rule_status = "UNDETERMINED"; rule_note = "Uniform Energy Factor is required but not provided"`
+          - otherwise, the storage volume is either greater than 100 gallons, or between 2 and 20, which is not covered by Appendix F: `else:`
+            - create a note indicating that gas water heaters greater with input capacity < 75,000 btu/hr and the given storage volume are not covered under Appendix F: `rule_note: "Gas water heaters with an input capacity < 75,000 btu/hr and a storage capacity of " + storage_volume + " are not covered under Appendix F"`
```

Comment:
Recommend updating note to "The storage tank volume is greater than 100 gallons and based on the size categories in ASHRAE 90.1 Table 7.8 and Appendix F it would not be classified as a gas storage water heater. Consequently, this rule was not able to be assessed for this service water heater. "